rllib/algorithms/sac/rnnsac.py

```diff
@@ -20,6 +20,7 @@
         # state outputs of the immediately preceding sequence).
         "zero_init_states": True,
         "replay_buffer_config": {
+            "storage_unit": "sequences",
```

Comment:
Can you add a comment here explaining the difference between "sequences" and "timesteps" for this particular buffer type? This would be very useful for users looking through the code and comparing e.g. DQN with R2D2.